Gemfile.lock

```diff
@@ -51,20 +51,10 @@ GEM
       tzinfo (~> 1.1)
     acts_as_list (0.9.15)
       activerecord (>= 3.0)
-    addressable (2.5.2)
-      public_suffix (>= 2.0.2, < 4.0)
     arel (6.0.4)
-    aruba (0.14.5)
-      childprocess (>= 0.6.3, < 0.10.0)
-      contracts (~> 0.9)
-      cucumber (>= 1.3.19)
-      ffi (~> 1.9.10)
-      rspec-expectations (>= 2.99)
-      thor (~> 0.19)
     ast (2.4.0)
-    autoprefixer-rails (8.6.3)
+    autoprefixer-rails (9.0.0)
```

Comment:
Could you revert `autoprefixer-rails` back to 8.6.3? I'd hate for an incompatibility caused by a major version bump to break the site.